sapphire/simulations/base.py

```diff
@@ -84,29 +84,32 @@ def run(self):
             chosen_core_pos = shower_parameters['core_pos']
             chosen_radius = np.sqrt( chosen_core_pos[0]**2. + chosen_core_pos[1]**2. )
 
-            if (chosen_energy < (13 + 0.1)) and (chosen_energy > (13 - 0.1)) and chosen_radius > 40:
```

Comment:
@kaspervd 
These if statements can be clarified:
```python
if (chosen_energy < (16.5 + 0.1)) and (chosen_energy > (16.5 - 0.1)) and chosen_radius > 500:
```
You can drop the brackets around the comparisons:
```python
if chosen_energy < (16.5 + 0.1) and chosen_energy > (16.5 - 0.1) and chosen_radius > 500:
```
And you can combine comparisons:
```python
if (16.5 - 0.1) < chosen_energy < (16.5 + 0.1) and chosen_radius > 500:
```
Or simply check the difference:
```python
if abs(chosen_energy - 16.5) < 0.1 and chosen_radius > 500:
```

And instead of comparing each energy value individually you can compare them all in one go to keep it DRY:
```python
energies = [13, 13.5, 14, ...] 
radiuses = [50, 65, 75, ...]
if any(abs(chosen_energy - energy) < 0.1 and chosen_radius > radius for energy, radius in zip(energies, radiuses)):
    continue
```